js/common/lib/inference-session.ts

```diff
@@ -430,11 +431,53 @@ export declare namespace InferenceSession {
 
   // #region value metadata
 
-  // eslint-disable-next-line @typescript-eslint/no-empty-interface
-  interface ValueMetadata {
-    // TBD
+  /**
+   * The common part of the value metadata type for both tensor and non-tensor values.
+   */
+  interface ValueMetadataBase {
```

Comment:
The naming convention seems to be ValueXxxMetadata, so maybe ValueBaseMetadata is better?

Reply:
How about NonTensorValueMetadata and TensorValueMeta, while keeping ValueMetadataBase? Anyway, it's not a big problem and feel free to ignore my comment.